core/src/main/java/discord4j/core/object/entity/Message.java

```diff
@@ -411,6 +411,8 @@ public Type getType() {
      *
      * @return A {@link Mono} where, upon successful completion, emits nothing; indicating the messaged was pinned. If
      * an error is received, it is emitted through the {@code Mono}.
+     * <br>
+     * The max pinned messages in a Text Channel actually is 50.
```

Comment:
If we can't find an official source mentioning this limit we should remove it. Otherwise we should include the link.

Reply:
The only official info about this is the Error Code from JSON.
https://github.com/discordapp/discord-api-docs/blob/master/docs/topics/Opcodes_and_Status_Codes.md#json-error-codes
Where the pinned messages are the code 30003

Reply:
Can you remove this then?

Reply:
Ok, maybe can add https://discordapp.com/developers/docs/topics/opcodes-and-status-codes#json in any class.. or well use this in any Exceptions info.